themes/ropensci/layouts/partials/email/share-url.html

```diff
@@ -0,0 +1 @@
+mailto:?subject={{ .Title  | replaceRE "&" "%26" }}{{ if ( eq .Type "commcalls" ) }} (community call){{ end }}, via rOpenSci&body={{ .Title }} {{ .Permalink }} via rOpenSci
```

Comment:
it is via rOpenSci not by rOpenSci because events are not necessarily by rOpenSci, if that makes sense.

Reply:
Yup, totally!

For the email text, what about putting via rOpenSci right after the title and then adding a line break before the link (otherwise it looks a bit jumbled in the email). Not sure if `<br>` is the right code for line break though!

```suggestion
mailto:?subject={{ .Title  | replaceRE "&" "%26" }}{{ if ( eq .Type "commcalls" ) }} (community call){{ end }}, via rOpenSci&body={{ .Title }} via rOpenSci <br> {{ .Permalink }}
```

Reply:
Let's try!😀